server/index.js

```diff
@@ -2,12 +2,15 @@ const express = require('express');
 const controllerQ = require('./controllers/questions');
 const reviews = require('./controllers/review.js');
 const product = require('./controllers/product.js');
+const cors = require('cors');
```

Comment:
What is this middleware for?

Reply:
It was for resolving an error when setting up post route